cropper/app/controllers/Application.scala

```diff
@@ -44,52 +50,30 @@ object Application extends Controller with ArgoHelpers {
 
   def index = Authenticated { indexResponse }
 
+  def export = Authenticated.async(parse.json) { httpRequest =>
+    httpRequest.body.validate[ExportRequest] map { exportRequest =>
```

Comment:
Note that I moved it from `Authenticated.async(parse.json[ExportRequest])` because that causes a nasty HTML page to be returned if the parsing failed, which is silly, we want a nice JSON error.